src/test/scala/com/databricks/spark/xml/XmlSuite.scala

```diff
@@ -21,14 +21,12 @@ import java.nio.file.Files
 import java.sql.{Date, Timestamp}
 
 import scala.io.Source
-
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.io.{LongWritable, Text}
 import org.apache.hadoop.io.compress.GzipCodec
 import org.scalatest.{BeforeAndAfterAll, FunSuite}
-
```

Comment:
This empty line is actually a style - https://github.com/databricks/scala-style-guide#imports :-)

Reply:
I fix it

Reply:
it will be better if idea will tip about it :)